build/css/index.css

```diff
@@ -0,0 +1,101 @@
+article,
+aside,
+body,
+caption,
+div,
+figure,
+footer,
+h1,
+h2,
+h3,
+header,
+hgroup,
+html,
+nav,
+p,
+pre,
+section,
+table,
+tbody,
+td,
+tfoot,
+th,
+thead,
+tr {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  font-family: "Open Sans", sans-serif;
+}
+
+.key {
+  font: bold 12px arial;
+  margin: 2px auto;
+  height: 50px;
+  outline: none;
+  background-image: -webkit-gradient(linear, left top, left bottom, from(#f4f1ee), to(#fff));
+  background-image: linear-gradient(top, #f4f1ee, #fff);
+  box-shadow: 0 8px 10px 0 rgba(0, 0, 0, 0.3), inset 0 4px 1px 1px white, inset 0 -3px 1px 1px rgba(204, 198, 197, 0.5);
+  -webkit-transition: all 0.1s linear;
+  transition: all 0.1s linear;
+  text-decoration: none;
+  text-align: center;
+  color: #83D244;
+  padding: 9px 0 0 15px;
+}
```

Comment:
no empty line between syles in some places.

Reply:
This file is created by scss compiler. Should I reconfigure it somehow?